site/ja/tutorials/load_data/images.ipynb

```diff
@@ -0,0 +1,3254 @@
+{
```

Comment:
<p>"以下は" is better than "これが" since "これが" can mean "the above code" </p>

 _Reply via <a href='https://app.reviewnb.com/tensorflow/docs/pull/427/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
<p>Exactly!</p>